.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "arrowParens": "avoid",
+  "tabWidth": 2,
+  "useTabs": false,
```

Comment:
Tabs are more accessible. ;) https://github.com/prettier/prettier/issues/7475

But I don't mind if you leave it like this.

Reply:
I would currently leave it as is and have a look at the linked issue so we can decide further down the road.